README.md

```diff
@@ -1,8 +1,30 @@
 # datastore-metadata
-All the LIS Datastore YAML files under one roof.
+All the LIS Datastore YAML files (and some others we consider "metadata") under one roof.
 
-This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore.
+This repository contains the description_[Genus]_[species].yml and README.[collection].yml files in the LIS Datastore as well as some other files handled as metadata.
+
+Several types of files should be checked for valid format and contents prior to pushing to the datastore-metadata repository:
+
+```
+  validate.sh description_genus_species YAML_FILE
```

Comment:
This script just kept printing the usage at me, I must be doing something wrong. But, I was able to validate the yaml with executable in the parent directory as per its usage example:
./validate readme.schema.json YAML_FILE